### ISSUE TITLE
Warnings in console when I exit to project manager

### ISSUE THREAD

**Issue by Torguen** [godotengine/godot#88538]
### Tested versions

v4.3.dev3.official [36e943b6b]

### System information

w10 64

### Issue description

I get some prompts in the console when I exit to the project manager but my project is all apparently fine.
I don't know if this is a bug?

![Captura5](https://github.com/godotengine/godot/assets/19652075/cc26aabc-6b2a-4480-af6a-caff5cf86b0c)

### Steps to reproduce

Import the project
Exit to project manager
See the console

### Minimal reproduction project (MRP)

[Test.zip](https://github.com/godotengine/godot/files/14328581/Test.zip)

**Comment by Torguen** [comment]
No idea about this?

**Comment by clayjohn** [comment]
This is a dev warning letting us know that certain things aren't being freed on exit of the project. It's nothing to worry about. 

For more context, are you using the Mobile or Forward+ renderer?

**Comment by AThousandShips** [comment]
The project manager uses compatibility right?

**Comment by Calinou** [comment]
> The project manager uses compatibility right?

Yes, unless forced to use something else with `--rendering-method` or `--rendering-driver`.

**Comment by Torguen** [comment]
> This is a dev warning letting us know that certain things aren't being freed on exit of the project. It's nothing to worry about.
> 
> For more context, are you using the Mobile or Forward+ renderer?

Mobile use
